### PR TITLE
[fix] Disabled HTTP query logging #1

### LINKED ISSUE #1: [change] Disable HTTP query logging
HTTP query logging is kind of bugged, it consumes a huge amount of space: https://github.com/influxdata/influxdb/issues/21726.

This is how I'm coping with it:

- disabling it in `/etc/influxdb/influxdb.conf` (`[http]`) section.
- Modify `/etc/logrotate.d/rsyslog` to add `maxsize 65M` to both sections in that file

I think we should do the same here.

### PR DESCRIPTION
**Pull request by pandafy**
Closes #1

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -43,6 +50,27 @@
...
+        path: /etc/logrotate.d/rsyslog
+        firstmatch: true
+        line: '        maxsize 65M'
+        insertafter: '\{'
+    - lineinfile:
+        path: /etc/logrotate.d/rsyslog
+        line: '        maxsize 65M'
+        insertafter: '\{'
+  tags: [influxdb1]
+
```
Comment (pandafy) [inline tasks/main.yml:73]: @nemesisdesign are you sure we need this change here? 

> Modify /etc/logrotate.d/rsyslog to add maxsize 65M to both sections in that file

Comment (nemesisdesign) [reply tasks/main.yml]: yes because the logged queries end up in these files even though they're disabled

File: tasks/main.yml [hunk 2]
```diff
@@ -43,6 +50,50 @@
...
+        line: '  log-enabled = false'
+        insertafter: '\[http\]'
+        state: absent
+    - lineinfile:
+        path: /etc/influxdb/influxdb.conf
+        line: '  query-log-enabled = false'
+        insertafter: '\[data\]'
+        state: absent
+  when: ansible_ow_influxdb_http_logging is true
+  tags: [influxdb]
```
Comment (pandafy) [inline tasks/main.yml:81]: This exists to revert the changes of previous task, otherwise it won't change anything if users first run with default settings and then realises that they forget to set `ansible_ow_influxdb_http_logging` to `true`.

File: README.md [hunk 3]
```diff
@@ -3,3 +3,12 @@
...
 
 Designed to be used in OpenWISP installations.
+
+Role Variables
+==============
+
+```yaml
+# By default HTTP query logging of InfluxDB is turned off since it can occupy a
+# lot of space. You can turn it on by setting below variable to true
+ansible_ow_influxdb_http_logging: true
```
Comment (nemesisdesign) [inline README.md:13]: I would call this simply `influxdb_query_logging` and would refer to it as query logging instead of HTTP query logging since we're managing both HTTP logging and query logging, what do you think?

**Review by nemesisdesign — APPROVED** [review]
